src/backend/distributed/planner/multi_logical_optimizer.c

```diff
@@ -2408,19 +2388,18 @@ ProcessHavingClauseForWorkerQuery(Node *originalHavingQual,
 		 * If the GROUP BY or PARTITION BY is not on the distribution column
 		 * then we need to combine the aggregates in the HAVING across shards.
 		 */
-		WorkerAggregateWalkerContext *workerAggContext = palloc0(
-			sizeof(WorkerAggregateWalkerContext));
+		WorkerAggregateWalkerContext workerAggContext = { 0 };
 
-		workerAggContext->extendedOpNodeProperties = extendedOpNodeProperties;
-		workerAggContext->expressionList = NIL;
-		workerAggContext->createGroupByClause = false;
+		workerAggContext.extendedOpNodeProperties = extendedOpNodeProperties,
```

Comment:
Why does this line end in a comma? Is that on purpose?

Reply:
Leftover from moving away from struct initializer syntax